CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to AgnosUI
+
+We are always looking for quality contributions and will be happy to accept your Pull Requests.
+
+AgnosUI is a powerful library that provides configurable Typescript component factories, headless wrappers and ready-to-use components compliant to the [CSS Bootstrap design](https://getbootstrap.com/).
+
+The [Open Source Guides](https://opensource.guide/) website has a collection of resources for individuals, communities, and companies. These resources help people who want to learn how to run and contribute to open source projects. Contributors and people new to open source alike will find the following guides especially useful:
+
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+- [Building Welcoming Communities](https://opensource.guide/building-community/)
+
```

Comment:
Shouldn't include the minimum expectation for a contribution?
For example, if someone wants to contribute a new component (very corner use case but...), should it include core + all frameworks? headless + bootstrap? same for bugs /issues, do we accept a PR for only a specific framework?